modules/tasks/tasks/src/main/java/org/motechproject/tasks/domain/mds/task/DataSource.java

```diff
@@ -32,6 +32,7 @@
     private Long objectId;
     private String type;
     private String name;
+    private String specifiedName;
```

Comment:
It is possible however it will take a lot of time, because on the frontend, objects action and datasource have already different field named 'displayName'